### DISCUSSION TITLE
Enabling WiFi

### DISCUSSION THREAD

**Discussion by tony1tf** [cardonabits/haxo-hw#58] [General]
I have tried two RPi Zero W's and the haxophone image doesn't seem to allow the WiFi to be enabled. 
I had put the wpa_supplicant file on the boot sector previously and it doesn't appear in the /etc directory, so I manually edited it with a keyboard and HDMI monitor plugged in, logged in as 'pi' with 'haxophone' pwd.
Commands like ifconfig wlan0 up 
reports wlan0: ERROR while getting interface flags: No such device
The second R Pi Zero W is not even connected to the haxophone HAT as it doesn't yet have a 40 pin connector.
I know the wifi is working properly on at least one of these Pi's with a previous SD card.
Not sure where to go from here.
Tony

**Comment by jcard0na** [comment]
Hi @tony1tf,

The images I release have wifi disabled to optimize for audio latency.  You can re-enable it by commenting out 
```
dtoverlay=disable-wifi
```
in `/boot/config.txt`
For that you'll need to disable overlay and make the boot partition writable.  Both can be done via `raspi-config`
```
sudo raspi-config
Performance Options -> Overlay Filesystem -> Enable Overlay? No -> Make Boot Filesystem Writeable? Yes
```

**Reply by tony1tf** [reply]
Thanks for your reply, but things have got worse, even before I start changing the overlay - I downloaded a new image from https://bit.ly/haxo-rpiz and I am having problem on boot up - with the Haxophone connected or not - the boot sequence gets to "Reached target Sound Card" and then hangs for a while before outputting more console messages until it reboots.  I disconnected the Haxophone, in case attaching the HDMI adaptor to the Pi Zero which presses against the sound o/p socket had flexed the pcb and damaged something, but this made no difference. I have filed some of the plastic off the HDMI adaptor to make it fit better against the sound o/p socket. The Haxophone doesn't produce any sound out now, even with no display or keyboard connected, and I can no longer see the several flashes of the green LED as the software is enabled. I am helpless, because I never get to see a command prompt before it reboots. I haven't got the image that worked previously because the images are too large to store more than one on my old MacBook.
Also, I can only see the image link in one of the posts on this discussion group - nowhere else.
There's another newer link in the software discussion with @RyanParsley https://bitly.com/rpi-zero, but that link is broken. 
Tony
